package.json

```diff
@@ -38,6 +38,8 @@
   "author": "ZEISS Digital Innovation Partners",
   "license": "MIT",
   "dependencies": {
+    "date-fns": "^2.0.0-beta.1",
+    "jest-environment-node": "^24.8.0",
```

Comment:
why this deps here?

Reply:
`date-fns` was there in 0.7.0 and removed in 0.7.1...looks like a merge issue.
I needed `jest-environment-node` for test:visual, but I didn't install it with `--save`, so I'm a bit surprised it's there.

Reply:
Please remove them

Reply:
Travis CI build fails because of:
`Error: Cannot find module 'jest-environment-node'`

Reply:
it's weird because it has been working before

Reply:
then let's add `jest-environment-node`, but please make sure it is in `devDependencies` and not in `dependencies`

Reply:
Well, I see why it has been broken. You've committed `package-lock.json` with new/updated versions of dependencies.

Reply:
So what can I do to fix it?

Reply:
Let's keep it in the deps list...